docs/dev/README.md

```
@@ -25,9 +37,25 @@ for local development, this dependency must be installed before *Testing Locally
-2. Run `make install` to install `CRD's`
-3. Export GIT_REPO_PATH. This path is used inside the operator container and goes not need to exist on the host. `export GIT_REPO_PATH=/git`
+2. Run `make install` to install CRDs
+3. Export GIT_REPO_PATH. This path is used inside the operator container and does not need to exist on the host. `export GIT_REPO_PATH=/git`
 4. Run `make run` to start the operator
    1. Or start the operator in your preferred manner
-5. Run `./docs/dev/seed.sh` to see all the configs/secrets in the cluster
-   1. Depending on what reconciler you are working on feel free to comment out anything in the file not related
\ No newline at end of file
+5. Run `./docs/dev/seed.sh` to seed all the configs/secrets in the cluster
+   1. This creates dummy secrets the operator needs: kubeconfig (to deploy test operators), GitHub token (for PR creation), Pyxis API key (for certification submission), Docker registry credentials, and SSH keys
+      1. Depending on how you want to execute the CI pipeline, depends on what secrets need to be created in the cluster, all execution methods can be found [here](https://github.com/redhat-openshift-ecosystem/certification-releases/blob/main/4.9/ga/ci-pipeline.md#execute-the-pipeline-development-iterations)
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)